content/messaging/agent-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -5315,7 +5315,7 @@ Informs about the message a Customer is currently typing. The push payload conta
   "chat_id": "PJ0MRSHTDG",
   "thread_id": "K600PKZON8",
   "sneak_peek": {   
-    author_id: "d17cd570-11a9-45c0-45c0-1b020b7586dc",
+    author_id: "b7eff798-f8df-4364-8059-649c35c9ed0c",
```

Comment:
Why isn't it a full email here?

Reply:
sneak peeks are generated by customers so `author_id == customer_id`